docs/madr/decisions/076-include-envoy-resource-name-tp-in-layout-endpoint.md

```diff
@@ -0,0 +1,129 @@
+# Include Envoy resource name and transparent proxy configuration in `_layout` endpoint 
+
+* Status: accepted
+
+Technical Story: https://github.com/kumahq/kuma/issues/13847
+
+## Context and Problem Statement
+
+While working on naming resources based on KRI ([MADR](https://github.com/kumahq/kuma/pull/13485/files)), 
+we have discovered that we are missing certain information in Inspect API,
+precisely in `_layout` endpoint. Which was designed in [Inspect API redesign MADR](https://docs.google.com/document/d/1EzZpk3wwneIxQNPK7WXJqhW3Y3CS1AWMXIcXix9LEoc/edit?tab=t.g7ooo2ntj4jj)
+
+Right now schema for `_layout` endpoint look like this:
+
+```yaml
+kri: <dataplane_resource_identifier>
+name: <dataplane_name>
+labels: <dataplane_labels>
+inbounds:
+  - kri: <resource_identifier> # Inbound KRI
+    port: <port_number> # port number
+    protocol: http # port protocol 
+outbounds:
+  - kri: <resource_identifier> # MS/MMZS/MES resource identifier
+    port: <port_number> # MS/MMZS/MES exposed port number
+    protocol: TCP
+  - kri: <resource_identifier> # MS/MMZS/MES resource identifier
+    port: <port_number> # MS/MMZS/MES exposed port number
+    protocol: HTTP
+```
+
+While trying to name Envoy resources using KRI, we've discovered that this will drastically increase the cardinality of metrics.
+We want to generate a simplified name for inbound that will look like: `<common_prefix>_sectionName`, with this new name
+it will be hard for GUI or other API users to correlate information from `_layout` endpoint with Envoy resources or stats.
+
+### Passthrough inbound/outbound
+
+While using transparent proxy, we add passthrough inbound and outbound which does not fit our current inbound/outbound schema
+since it does not have KRI it is not a real resource and cannot be targeted in policies. We need to include these extra inbound/outbound
+in our current `_layout` endpoint.
+
+## Design
+
+### Generated inbound name
+
+To address the issue with simplified naming on inbounds for Envoy resources and stats, we want to add this name to existing inbounds in api.
+There is question on how to name this field, here are the ideas:
```

Comment:
It would be easier for the GUI if this field was also present in the outbound. It also gives us a possible escape hatch for cases where it might differ from KRI in the future, even if both fields have the same KRI value for now.

cc. @johncowen

Reply:
Sounds good 👍 

This is the thing/comment that led us here:

> anyone can just do `statsResponseParsedIntoObject[inbounds[num].proxyResourceName].downstream_cx_active` or `statsResponseParsedIntoObject[outbounds[num].proxyResourceName].downstream_cx_active` consistently